kratos/includes/mortar_classes.h

```diff
@@ -902,6 +948,54 @@ class DerivativeData
     ///@name Private Operations
     ///@{
 
+    ///@}
+    ///@name Serialization
+    ///@{
+
+    friend class Serializer;
+
+    virtual void save(Serializer& rSerializer) const
+    {
+        rSerializer.save("PenaltyParameter", PenaltyParameter);
+        rSerializer.save("ScaleFactor", ScaleFactor);
+        rSerializer.save("NormalSlave", NormalSlave);
+        rSerializer.save("NormalMaster", NormalMaster);
+        rSerializer.save("X1", X1);
+        rSerializer.save("u1", u1);
+        rSerializer.save("X2", X2);
```

Comment:
Are all of those members?

Reply:
Yes, but if I change the name all my developments will stop working